nmmo/task/base_predicates.py

```diff
@@ -43,6 +43,11 @@ def StayAlive(gs: GameState,
   """True if all subjects are alive.
   """
   return count(subject.health > 0) == len(subject)
+  # The below is for speed testing (bypass GroupView)
```

Comment:
let's not include it in this PR, perf testing can go in a different PR if we want to commit it at all

Reply:
Removed!